homeassistant/components/zwave_js/device_trigger.py

```diff
@@ -184,6 +192,26 @@
 )
 
 
+def copy_available_params(
+    input_dict: dict, output_dict: dict, params: list[str]
```

Comment:
Are the keys in the input and output dicts always strings? Then we can make the typing more specific.